EIPS/eip-1.md

```diff
@@ -119,6 +119,8 @@ Each EIP must begin with an [RFC 822](https://www.ietf.org/rfc/rfc822.txt) style
 
 ` title:` *EIP title*
 
+` description:` *Provide a simplified and layman-accessible explanation of the EIP.*
```

Comment:
It was ERC20 tokens send to my luno ethereum wallet. Was stuck on etherscan and they released the tokens for public.